cypress/integration/search/post_search_display.js

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/*eslint max-nested-callbacks: ["error", 3]*/
+
+describe('Post search display', () => {
+    it('S14252 After clearing search query, search options display', () => {
+        // # Login and navigate to the app
+        cy.apiLogin('user-1');
+        cy.visit('/');
+        const searchWord = 'Hello';
+
+        // # post message
+        cy.postMessage(searchWord);
+
+        // # search word in searchBox and validate searchWord
+        cy.get('#searchBox').type(searchWord).type('{enter}').should('have.value', searchWord);
+
+        // # click on "x" displayed on searchbox
+        cy.get('#searchClearButton').click();
+
+        // # RHS should be visible with search results
+        cy.get('.sidebar-right__body').should('be.visible');
```

Comment:
Please add id as `searchResultsContainer` to [SearchResults](https://github.com/mattermost/mattermost-webapp/blob/5363819630d9a208c9259235b6192d70a94765d6/components/search_results/search_results.jsx#L213) so it can be:
```suggestion
        cy.get('#searchResultsContainer').should('be.visible');
```